programs/programs/urgent_needs/ma/free_tax_help.py

```diff
@@ -0,0 +1,25 @@
+from ..base import UrgentNeedFunction
+
+
+class FreeTaxHelp(UrgentNeedFunction):
+    """
+    CEOC - Free Tax Help (VITA)
+    Free tax preparation to help residents file returns and claim credits.
+    Cambridge resident or income-eligible. Generally ≤ $60,000 annual income
+
+    """
+
+    dependencies = ["county", "income_amount", "income_frequency"]
+    eligible_city = "Cambridge"
+    income_limit = 60000
+
+    def eligible(self):
```

Comment:
Can we add a type hint here? I've noticed updates in recent PRs where the returned type was accidentally changed. I think this is something we should be more intentional about.

```python
def eligible(self) -> bool:
```

(Totally not your fault BTW. I know this pattern exists in other places)

Nit: We should also remove the blank line below the method signature.